query-planner-wasm/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@apollo/query-planner-wasm",
+  "version": "0.0.3",
```

Comment:
Should this be...
```suggestion
  "version": "0.0.4",
```

Reply:
Or maybe you've already done a release elsewhere that will merge into this branch.

Reply:
Nope, you're right!